test/functorch/test_control_flow.py

```diff
@@ -256,6 +256,109 @@ def false_fn(x):
         pred = torch.tensor(False, device="cuda")
         result = cond(pred, true_fn, false_fn, [x])
         self.assertEqual(result, torch.cos(x))
+        
```

Comment:
For each of these added tests, can you add the following test to check the graph:
```python
# Given following f
def f(...):
   result = forward_with_cond
   grad_out = torch.ones_like(result)
   return torch.autograd.grad(result, (x,), grad_out)

# can use make_fx to capture the graph
gm = make_fx(f)(...)
# check the graph
self.assertExpectedInline(gm.print_readable().strip(), """hahaha""")
```
You can use `EXPECTTEST_ACCEPT=1 python test/functorch/test_control_flow.py` to automatically populate the expected inline string.

Reply:
I added the graph checks as you suggested.